FitNesseRoot/FitNesse/UserGuide/AdministeringFitNesse/ConfigurationFile/content.txt

```diff
@@ -177,6 +177,7 @@ There are more properties which can be used to tweak parts of FitNesse:
  * '''VersionsController.days''' - number of days to keep old page versions around when using the Zip file based versions controller.
  * '''test.history.days''' - The number of days to keep test results around. Cleaned up after a new test run.
  * '''test.history.path''' - Location to store the test results. The default location is ''!-FitNesseRoot-!/files/testResults''.
+ * '''TestHistory.purgeTime''' - A comma separated list of numbers for purge options on the ''Test History'' page.
```

Comment:
Can you update the description to give an indication what the numbers represent?
For instance:
```suggestion
 * '''TestHistory.purgeTime''' - A comma separated list of the age, in number of days, to offer as purge options on the ''Test History'' page.
```